sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfile.cs

```diff
@@ -74,7 +74,7 @@ public SharingProfile(string permissions = default(string), IList<SharingProfile
         /// shared to community.
         /// </summary>
         [JsonProperty(PropertyName = "communityGalleryInfo")]
-        public object CommunityGalleryInfo { get; set; }
+        public CommunityGalleryInfo CommunityGalleryInfo { get; set; }
```

Comment:
It's nice to change the type from object to the specific type CommunityGalleryInfo, but it is still a breaking change.

Reply:
@Yao725 , and the breaking change was approved in the Swagger PR by the breaking change review board. Mostly because the functionality was broken without these changes. 
I am unaware of any customizations that can be done to mitigate these different breaking changes. If you have any advice, please let us know.